### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by AtaZh**
If you can also tell me that these changes will soon appear in the app itself, I'll be happy :)

### CODE REVIEW

**Comment by misaochan** [comment]
Hi @AtaZh ,

Thanks for submitting this! I'm away from my desktop for now, but I will test and merge as soon as I can. If everything goes well, I should hopefully be able to release an update with your changes in a week or so.

File: app/src/main/res/values-uk/strings.xml [hunk 1]
```diff
@@ -62,32 +72,46 @@
...
+  <string name="share_license_summary">Це зображення буде ліцензоване згідно з %1$s</string>
+  
   <string name="menu_download">Завантажити</string>
   <string name="preference_license">Ліцензія</string>
-  <string name="license_name_cc_by_sa">CCu00A0Attribution-ShareAlikeu00A03.0</string>
-  <string name="license_name_cc_by">CCu00A0Attributionu00A03.0</string>
+  <string name="use_previous">Використати попередню назву й опис</string>
+ 
+  <string name="allow_gps">Автоматично отримати поточне розташування</string>
+  <string name="allow_gps_summary">Отримати поточне розташування, щоб з'явилися підказки категорій, якщо зображення не має геотегів</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values-uk/strings.xml]: Please replace `з'явилися` with `з\'явилися`, thanks! Otherwise it might crash at runtime.

Comment (AtaZh) [inline app/src/main/res/values-uk/strings.xml]: Sorry, I forgot to escape apostrophes.

**Comment by nicolas-raoul** [comment]
Thanks for fixing :-)

**Comment by misaochan** [comment]
Hi @AtaZh ,

I've pushed an update with your fix, it should be live in several hours' time. Cheers!

**Pull request merged**